x-pack/plugins/cloud_security_posture/common/schemas/csp_configuration.ts

```diff
@@ -7,8 +7,10 @@
 import { schema as rt, TypeOf } from '@kbn/config-schema';
 
 export const cspRulesConfigSchema = rt.object({
-  activated_rules: rt.object({
-    cis_k8s: rt.arrayOf(rt.string()),
+  data_yaml: rt.object({
```

Comment:
Maybe worthwhile to add a comment here that this key `data_yaml` must always match the corresponding configuration key in cloudbeat.